ibis/backends/duckdb/__init__.py

```diff
@@ -99,18 +117,16 @@ def _s3(full_path, table_name=None):
 
     dataset = ds.dataset(full_path)
     table_name = table_name or _name_from_dataset(dataset)
-    quoted_table_name = _quote(table_name)
-    return quoted_table_name, dataset
+    return table_name, dataset, []
 
 
 @_generate_view_code.register(r".+", priority=1)
-def _default(_, **kwargs):
+def _default(path, **kwargs):
     raise ValueError(
-        """
-Unrecognized filetype or extension.
-Valid prefixes are parquet://, csv://, s3://, or file://
+        """Unrecognized file type or extension: {path}.
```

Comment:
The `path` variable is not interpolated at the moment.


```suggestion
        f"""Unrecognized file type or extension: {path}.
```